pkg/operation/botanist/component/logging/kuberbacproxy/kube_rbac_proxy.go

```diff
@@ -42,7 +42,7 @@ const (
 	PromtailTokenSecretName = promtailName
 )
 
-// Values are the values for the kube-rbac-proxy.
+// Values are the values for the kube-rbac-proxy and the event-logger
```

Comment:
```suggestion
// Values are the values for the kube-rbac-proxy.
```


Can you also adapt the `func NewKubeRBACProxy(so *Values) (component.Deployer, error) {` to be called simply `New` and any other function/variable names that contain duplicates of the package name.

Reply:
Sorry, but I can't do it for other functions/variables because this component consists of two other components `kube-RBAC-proxy` and `Promtail` and I insist to differentiate somehow between them.